src/ToggleButton.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { Box, FormControlLabel, Switch } from '@mui/material';
+import { useState } from 'react';
```

Comment:
Is there a particular reason why you explicitly exported `useState` from React if you already imported everything? Does it not let you use that variable if you imported everything from React in line 1?

Reply:
Yeah, when I delete useState it says that it can't find the name for it.